beacon/endpoints/rest/__init__.py

```diff
@@ -139,7 +139,7 @@ class GVariantsParameters(GVariantParametersBase):
                                   default='beacon-variant-annotation-v2.0.0-draft.2')
 
 
-def generic_handler(log_name, proxy, fetch_func, build_response_func):
+def generic_handler(log_name, by_entity_type, proxy, fetch_func, build_response_func):
```

Comment:
Make all the entity_type integers and not strings. Use for example an enum.
Strings are interned in Python and you never know when they'll be copied. Moreover, strings comparison is slower than _one_ integer comparison.

Reply:
I've replaced it with `IntEnum`.